pandas/_typing.py

```diff
@@ -1,6 +1,6 @@
 from dataclasses import dataclass
 from datetime import datetime, timedelta, tzinfo
-from io import IOBase
+from io import BufferedIOBase, RawIOBase, TextIOBase, TextIOWrapper
```

Comment:
I replaced the abstract class `IOBase` (it doesn't implement `write`) with its implementations.